pp/serv/serv.go

```diff
@@ -51,3 +62,32 @@ func startIPC() error {
 
 	return nil
 }
+
+func startHttpRPC() error {
+	http :=	newHTTPServer(rpc.DefaultHTTPTimeouts)
```

Comment:
better name it differently, it will easily mixed with http package if somehow we import that package later

Reply:
Do you mean the function name? How about startRpcServer()?

Reply:
Modified.